v1/acl.go

```diff
@@ -110,7 +115,7 @@ func (a *ACL) PostACLBootstrap(ctx context.Context) (*client.ACLToken, *WriteMet
 	return &final, meta, nil
 }
 
-func (a *ACL) GetACLTokens(ctx context.Context) (*[]client.ACLTokenListStub, *QueryMeta, error) {
+func (a *ACL) ListTokens(ctx context.Context) (*[]client.ACLTokenListStub, *QueryMeta, error) {
```

Comment:
```suggestion
func (a *ACL) Tokens(ctx context.Context) (*[]client.ACLTokenListStub, *QueryMeta, error) {
```
Same naming suggestion here.

Reply:
Done